repos/system_upgrade/el7toel8/actors/getinstalleddesktops/libraries/library.py

```diff
@@ -0,0 +1,17 @@
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRPM
+
+
+def get_installed_desktops():
+    api.current_logger().info("  Detecting desktop environments  ")
+    api.current_logger().info("==================================")
+
+    # Detect installed desktops by one of the base rpm packages
+    kde_desktop_installed = has_package(InstalledRPM, "kde-workspace")
+    gnome_desktop_installed = has_package(InstalledRPM, "gnome-session")
```

Comment:
This is not what it should do. Moreover, I cannot reproduce it. Could you provide additional info please (Specific RHEL version, installed packages etc.)?

Reply:
@Honza0297, it's because `gdm` (The GNOME Display Manager) is the **mandatory** package of the `kde-desktop` package group. 
For a proof, just look at the RHEL 7.x Server repo metadata, e.g.
```
<<URL to the RHEL 7.7 repo>>/Server/x86_64/os/repodata/6d132b085489dcfe4a0ee2b4c38d9578c38ab83cebe4df87e3a409b48e8fee57-comps-Server.x86_64.xml
```